pages/user/[username]/card.tsx

```diff
@@ -1,48 +1,167 @@
 import { GetServerSideProps, NextPage } from "next";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import { useTransition, animated } from "react-spring";
 import Button from "components/atoms/Button/button";
 import HeaderLogo from "components/molecules/HeaderLogo/header-logo";
-import DevCardCarousel from "components/organisms/DevCardCarousel/dev-card-carousel";
-import { STUB_DEV_CARDS } from "components/organisms/DevCardCarousel/stubData";
+import DevCardCarousel, { DevCardCarouselProps } from "components/organisms/DevCardCarousel/dev-card-carousel";
```

Comment:
nit: Will you clean up the imports where third-party imports (react, next, cntl, etc) are at the top, followed by local imports (not a blocker)

Reply:
Yah, i think i can turn this into a eslint rule too